docs/modules/datasources/pages/flow_datasource.adoc

```diff
@@ -108,3 +111,94 @@ Assuming that the `$node` variable is populated with a value from the `exporterN
 ```
 dscpOnExporterNodeAndInterface($node, $interface, $__from, $__to)
 ```
+
+=== applications(<limit>)
+
+The `aplications()` query returns a list of all available applications. This function is intended to be used inside the Filter function `withApplications()` as a template variable if needed.
+
+Currently this function accepts only a limit of records to be retrieved as argument if necessary.
```

Comment:
```suggestion
Currently this function accepts only a limit of records to be retrieved as argument if necessary.
```
Not clear on what is meant here. A "limited number of records"? It only retrieves records?